next.config.js

```diff
@@ -1,22 +1,27 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   images: {
-    domains: ["avatars.githubusercontent.com", "cdn.discordapp.com"],
+    domains: ['avatars.githubusercontent.com', 'user-images.githubusercontent.com', 'cdn.discordapp.com', 'i.ibb.co', 'i.postimg.cc', 'images2.imgbox.com', 'raw.githubusercontent.com', 'img.youtube.com'],
```

Comment:
Could you please explain this part?

Reply:
The Images in the projects.json come from these hosts. This to be replaced with normal image files in the document itself. Please see conversation in discord